hibernate-reactive-core/src/test/java/org/hibernate/reactive/configuration/ReactiveConnectionPoolTest.java

```diff
@@ -39,23 +39,21 @@
 import static org.hibernate.reactive.BaseReactiveTest.test;
 import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
 import static org.hibernate.reactive.containers.DatabaseConfiguration.getJdbcUrl;
-import static org.hibernate.reactive.testing.DBSelectionExtension.runOnlyFor;
 import static org.hibernate.reactive.testing.ReactiveAssertions.assertThrown;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @ExtendWith(VertxExtension.class)
 @TestInstance(TestInstance.Lifecycle.PER_METHOD)
 @Timeout(value = 10, timeUnit = TimeUnit.MINUTES)
+// We use native queries, which may be different for other DBs
+@EnableFor(POSTGRESQL)
 public class ReactiveConnectionPoolTest {
 
-	@RegisterExtension
-	public DBSelectionExtension dbSelection = runOnlyFor( POSTGRESQL );
-
 	@RegisterExtension
 	public TestingRegistryExtension registryExtension = new TestingRegistryExtension();
 
 	@RegisterExtension
-	public RunTestOnContext testOnContext = new RunTestOnContext();
+	public static RunTestOnContext testOnContext = new RunTestOnContext();
```

Comment:
Is this necessary? It doesn't seem related to this commit purpose

Reply:
no.. removed